File: 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/ui/client/hud/element/elements/NotifyType.java
package net.ccbluex.liquidbounce.ui.client.hud.element.elements;

import java.awt.*;

public enum NotifyType {
    OKAY(new Color(0,255,127),"check-circle"),
    WARN(new Color(255,75,0),"close-circle"),
    INFO(new Color(0, 160, 255),"information");

    public Color renderColor;
    public String icon;

    NotifyType(Color renderColor,String icon){
        this.renderColor=renderColor;
        this.icon=icon;
    }
}

File: src/main/java/net/ccbluex/liquidbounce/ui/client/hud/element/elements/notify/NotifyType.kt
package net.ccbluex.liquidbounce.ui.client.hud.element.elements

import java.awt.Color

enum class NotifyType(var renderColor: Color, var icon: String) {
    OKAY(Color(0, 255, 127), "check-circle"),
    WARN(Color(255, 75, 0), "close-circle"),
    INFO(Color(0, 160, 255),"information");
}


enum class FadeState { IN, STAY, OUT, END }